dbt_subprojects/dex/seeds/trades/otsea_base_base_trades_seed.csv

```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_month,block_date,block_time,block_number,token_sold_amount_raw,token_bought_amount_raw,token_sold_address,token_bought_address,maker,taker,project_contract_address,tx_hash,evt_index
+base,otsea,1,2024-05-01 00:00,2024-05-02 00:00,2024-05-02 07:41,13923182,88652482269503546099290,4000000000000000,0x9c1a0802fbe08bf7c68bb336820b7b2a7e4eb697,0x0000000000000000000000000000000000000000,0x9c1a0802fbe08bf7c68bb336820b7b2a7e4eb697,0xa836af59d35b3da3c9fb5cfd6f84d9e3bd8c5fd5,0xa836af59d35b3da3c9fb5cfd6f84d9e3bd8c5fd5,0xb6038c12bff786acb485fef9402d4215c97242a757c3f21ffebd0fe1aee7e92d,161
+base,otsea,1,2024-05-01 00:00,2024-05-09 00:00,2024-05-09 23:24,14253851,122404014850000000,1000000000000000000,0xf41c500e45868f7cf73098c5699c1343d6d30051,0x0000000000000000000000000000000000000000,0xf41c500e45868f7cf73098c5699c1343d6d30051,0xa836af59d35b3da3c9fb5cfd6f84d9e3bd8c5fd5,0xa836af59d35b3da3c9fb5cfd6f84d9e3bd8c5fd5,0x9d02c7614fc425df8614259e6b80890d13db5b8545a9095e3529d430fb6fc44f,226
```

Comment:
Please verify the trade:
https://basescan.org/tx/0x9d02c7614fc425df8614259e6b80890d13db5b8545a9095e3529d430fb6fc44f#eventlog
One trading token should be `0xfD0F44da2a76e61585032dE6E54174a4b0c1CfB8`.